### DISCUSSION TITLE
Formazione gruppi di lavoro - [CLOSED]

### DISCUSSION THREAD

**Discussion by mancusoa74** [ittagnelli/ats#1] [General]
juyciao, 
per iniziare effettivamente a lavorare su ATS abbiamo la necessità di suddividerci un pò in gruppi di specializzazione.
I gruppi possono cambiare nel tempo ed essere flessibili. In ogni caso tutti lavoriamo per lo stesso obiettivo, quindi non vediamo la divisione in gruppo come una limitazione, come una scusa per, ma come un opportunità per lavorare in gruppo e tutti insieme raggiungere l'obiettivo finale.

Detto ciò propongo di creare i seguenti gruppi:

1. **Gruppo UX**: gruppo User eXperience di grafici e fotografi
2. **Gruppo Dev**: sviluppo HTML, CSS e JS
3. **Gruppo Blog**: blog e contenuti del sito

Inizierei con questi tre gruppi. Se avete proposte per altri gruppi ok.

Datemi la lista dei componenti dei 3 gruppi ***entro domani COB (Close of Business)***

**Comment by Giacomo-Lurgo** [comment]
Buonasera professore, ecco i gruppi di specializzazione.

**Gruppo** **Ux**:  
 - Apetroaie Antonio
 - Viteritti William
 - Bortoluzzi Luca
 - Lurgo Giacomo
 - Chindris Alexandru
 
**Gruppo** **Dev**: 
 - Zaina Francesco
 - Pomba Federico
 - Garabello Marco
 - Laudani Simone
 - Quijada Pierpaolo
 - Ruggieri Dario
 
**Gruppo** **Blog e  Contenuti**:
 - Siracusa Michele
 - Solavagione Matteo
 - Sola Matteo
 - Sciannimanica Cristian
 - Chindris Alexandru

**Reply by mancusoa74** [reply]
molto bene!!

**Comment by mancusoa74** [comment]
ho aggiunto Alex al gruppo Blog e Contenuti

**Comment by mancusoa74** [comment]
I gruppi di lavoro che avete costituito si sono rivelati inefficaci, pertanto da oggi 24/10/2021 non sono più da considerarsi validi
